src/bg/user-script-registry.js

```diff
@@ -239,28 +239,26 @@ window.onUserScriptUninstall = onUserScriptUninstall;
 async function saveUserScript(userScript) {
   if (!(userScript instanceof EditableUserScript)) {
     throw new Error(
-        'Cannot save this type of UserScript object:'
+        'Cannot save this type of UserScript object: '
         + userScript.constructor.name);
   }
 
   function onSaveError(error) {
     let message;
     if (error.name == 'ConstraintError') {
       // Most likely due to namespace / name conflict.
-      message = chrome.i18n.getMessage(
-          'User script save failed: script named $1 already exists in namespace $2.',
-          [JSON.stringify(userScript.name),
-           JSON.stringify(userScript.namespace)]);
+      message = _('User script save failed: script named $1 already exists in namespace $2.',
+                  JSON.stringify(userScript.name),
```

Comment:
All continued statements should be indented by exactly four spaces.

Reply:
Tried that, was ugly, but no problem, it's okay...